### DISCUSSION TITLE
[Help] Amazon Linux 2023

### DISCUSSION THREAD

**Discussion by jcmaciel** [libvips/libvips#4029] [General]
Hello everyone, I would like some help using libvips on AWS using Amazon Linux 2023. 

I checked in the documentation that the package is not supported, how can I use it?

I need this dependency for my Ruby on Rails 7.1 application.

Thanks in advance,

**Comment by lovell** [comment]
https://github.com/amazonlinux/amazon-linux-2023/issues/295

**Reply by jcmaciel** [reply]
Thanks @lovell i see this issue, but no updates from AWS Team..
